frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.component.tsx

```diff
@@ -15,7 +15,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Not sure how important this is, but when I open a ticket (from the list, from the URL doesn't seem to happen) with a default pin, the latter is "selected" even if the pin does not exist
![image](https://github.com/3drepo/3drepo.io/assets/43547703/7ad64b1b-5255-4668-84a7-205935e8ef04)